lib/methods/get-orders/response-builder.js

```diff
@@ -0,0 +1,24 @@
+const buildGetOrderssResponse = response => {
+  const getItem = item => ({
```

Comment:
This function could be outside of the `buildGetOrdersResponse` function scope. Could you move it?